### PR TITLE
updated README.md file with Labs 37 contributors

### PR DESCRIPTION
**Pull request by alvin-joseph**
# Description

Updated Readme file with Labs 37 contributors

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by nickcabuno — APPROVED** [review]
go frontend team wooo

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ Our team is developing an interactive map that identifies potential instances of
...
+|[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/alvin-joseph031/)|[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/davis-cooper-jackson/)|[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/devin-mitchell100/)|[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/jossters/)|[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/ryan-dill-65398610b/)|[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/tony-yang-b3318985/)|[ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/in/nick-cabuno/)|
+
```
Comment (JAaron93) [inline README.md:18]: Consistent formatting with the last cohort, Labs36. Looks good!

**Review by JAaron93 — APPROVED** [review]
Looks like you got everyone's GitHub and LinkedIn accounts embedded here. Good work!

**Review by RyderBlues — APPROVED** [review]
wew~

**Pull request merged**